content/docs/guide/content.md

```diff
@@ -146,6 +146,8 @@ The `store` command offers a variety of useful flags:
   backend failures.
 - `-x, --excludes` lets you choose to exclude files and folders from the
   snapshot.
+- `--pedantic` will exit knoxite after the first erroneous operation on a
```

Comment:
Sounds too clunky, imo: we simply exit after the first error occurred, even if it's not a file/folder operation.